doc/ledger3.texi

```diff
@@ -6702,9 +6702,15 @@ Record the hash of each transaction in a @var{Hash} metadata value,
 according to the hashing algorithm given by the @var{ALGO}
 argument. Note that if a @var{Hash} metadata value is explicitly
 provided and does not match what would have been generated, an error is
-reported. Hashes depend on previous entries, such that setting a single
-hash value is sufficient to guarantee the shape of the entire history
-leading up to that entry.
+reported. This provided value can also just be a prefix, at whatever
+length the user wishes, in which case only this amount of text is
+compared to the generated hash. Printing the entry with hashes enabled
```

Comment:
On the surface this seems like a complexity that discourage users - if it's unavoidable, perhaps say a little more about the motivation and recommendations for use ? Would a different default behaviour work instead ?

Reply:
The motivation is answered in the previous comment... perhaps it should be worded differently here?

Reply:
Thanks, I've added this and more to the Texinfo file.